lib/_header.erb

```diff
@@ -18,10 +18,22 @@
                     <%- doc_link = get_github_link(@site_map['Documents'][0].values[0][0]["Link"].dup) -%>
                 <%- end -%>
 
-                <a href="<%= doc_link -%>" class="doc-actions__item doc-actions__item--web">
-                    <img src="img/icons/edit.svg" alt="">
-                    <span>Suggest an Edit</span>
-                </a>
+                <div class="dropdown doc-actions__edit">
+                    <a href="" class="dropdown__toggle dropdown__toggle--caret doc-actions__item">
+                        <img src="img/icons/edit.svg" alt="">
+                        <span>Suggest an Edit</span>
+                    </a>
+
+                    <div class="dropdown__menu dropdown__menu--right">
+                        <ol>
+                            <li>Log into Github.</li>
+                            <li>Edit files with your changes by clicking on 'Edit the file in your fork of this project' button in Github.</li>
+                            <li>Commit changes via 'Create a new branch for this commit and start a pull request'.</li>
+                        </ol>
+
+                        <a href="https://github.com/minio/gluegun/blob/master/README.md" class="button button--primary button--block">Edit in Github</a>
```

Comment:
This link should be dynamic instead of being hardcoded, so we can just replace the value of the href attribute with `<%= doc_link -%>`